From 1d1cc65ce6d306f548e725718c8c02d2fec5daac Mon Sep 17 00:00:00 2001
From: Tong Zhang <zhangt@frib.msu.edu>
Date: Thu, 16 Mar 2017 13:16:36 -0400
Subject: [PATCH] Fix - wrong version.

- should be 0.2.1.
---
 phantasy/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/phantasy/__init__.py b/phantasy/__init__.py
index 541c772..4ff25f1 100644
--- a/phantasy/__init__.py
+++ b/phantasy/__init__.py
@@ -24,7 +24,7 @@ from phantasy.library import *
 __authors__ = "Tong Zhang"
 __copyright__ = "(c) 2016, Facility for Rare Isotope beams, Michigan State University"
 __contact__ = "Tong Zhang <zhangt@frib.msu.edu>"
-__version__ = "0.2.0"
+__version__ = "0.2.1"
 
 __doc__ = """PHANTASY:
 Physics high-level applications and toolkits for accelerator system.
-- 
2.7.4

